src/components/Tooltip/Tooltip.js

```diff
@@ -13,9 +13,10 @@ const propTypes = {
   hideShortcut: PropTypes.bool,
 };
 
-const Tooltip = forwardRef( ({ content = '', children, hideShortcut }, forwardedRef) => {
+const Tooltip = forwardRef( ({ content = '', children, hideShortcut }, ref) => {
   const timeoutRef = useRef(null);
-  const childRef = forwardedRef ? forwardedRef : useRef(null);
+  const childRef = useRef(null);
+  useImperativeHandle(ref, () => childRef.current);
```

Comment:
Fixes conditional use of `useRef` (shouldn't conditionally use hooks, can mess with the flow although in this case it's dependent on ref so probably safeish). Also, this is the proper way to use a ref internally but also have it output with `forwardRef` (unfortunately also not very well documented).

Reply:
Ah this is pretty cool. Thanks for showing this.